data/transactions/logic/evalStateful_test.go

```diff
@@ -2537,17 +2538,17 @@ func TestLatestTimestamp(t *testing.T) {
 func TestBlockSeed(t *testing.T) {
 	ep, txn, l := makeSampleEnv()
 
-	// makeSampleENv creates txns with fv, lv that don't actually fit the round
+	// makeSampleEnv creates txns with fv, lv that don't actually fit the round
 	// in l.  Nothing in most tests cares. But the rule for `block` is
 	// related to lv and the current round, so we set the fv,lv more
 	// realistically.
 	txn.FirstValid = l.round() - 10
 	txn.LastValid = l.round() + 10
 
 	// l.round() is 0xffffffff+5 = 4294967300 in test ledger
-	testApp(t, "int 4294967299; block BlkSeed; len; int 32; ==", ep) // current - 1
-	testApp(t, "int 4294967300; block BlkSeed; len; int 32; ==", ep,
-		"not available") // can't get current round's blockseed
+	testApp(t, "int 4294967300; block BlkSeed; len; int 32; ==", ep)
```

Comment:
this is confusing.
cow.Round() is always the current (latest +1). So the old test is valid and the new one is not.

Reply:
my understanding is we supply cow to contracts and ledger to logicsig. But the same `Round()` method returns the different values.
Maybe one should be Latest() and another Round()?

Reply:
I think I know what's going on now.  I added a Round() method to ledger.Ledger, so I could use it as a LedgerForSignature.  I should have used Latest()+1, but used Latest(). I need to get them doing the same thing, and then the test will go back to what it was.

Reply:
This should be back to making sense.